package.json

```diff
@@ -342,7 +342,6 @@
   "dependencies": {
     "bluebird": "^3.3.0",
     "body-parser": "^1.16.1",
-    "browserify": "^14.5.0",
```

Comment:
I think moving it to `devDependencies` caused issue for me. I wanted to use karma from certain commit in package.json:

`"karma": "https://github.com/karma-runner/karma#018e6bec1775d4a6442e2554e510a989c8f0d3be"`

and I got error:

`Front-end script compile failed with error: Error: Cannot find module 'browserify'` when launching unit tests. 

To fix it I needed to install browserify: `npm install browserify`.